UndertaleModLib/UndertaleChunks.cs

```diff
@@ -900,6 +900,19 @@ public class UndertaleChunkCODE : UndertaleListChunk<UndertaleCode>
     {
         public override string Name => "CODE";
 
+        private void CheckForGM2023_8(UndertaleReader reader)
+        {
+            if (!reader.undertaleData.IsVersionAtLeast(2023, 2) || reader.undertaleData.IsVersionAtLeast(2023, 8))
+                return;
+
+            // Check for a `pushref` instruction
+            if (List.Any(c => c.Instructions.Any(x => x.Kind == UndertaleInstruction.Opcode.Break
+                                                      && x.Value is short val && val == -11)))
+            {
+                reader.undertaleData.SetGMS2Version(2023, 8);
```

Comment:
i assume its not possible for a chunk to overwrite this later?

Reply:
WDYM?
After this line, no other GM check should be able to downgrade it if you meant that

Reply:
Ah ok. cool